burr/tracking/client.py

```diff
@@ -85,7 +85,10 @@ def __init__(
 
     @classmethod
     def get_storage_path(cls, project, storage_dir) -> str:
-        return os.path.join(os.path.expanduser(storage_dir), project)
+        # need to replace `:` with `_` for the project_id; it becomes part of the path and characters
```

Comment:
I don't think this is the right solution. Better to error if it doesn't match. We can fix upstream, or add a patch to have `demo_` instead of `demo:`